File: 2819-remove-trailing-zeros-from-a-string/2819-remove-trailing-zeros-from-a-string.java
class Solution {
    public String removeTrailingZeros(String num) {
           int index= num.length()-1;
        for(int i=num.length()-1;i>=0;i--){
            char s= num.charAt(i);
            if(s!='0'){
                 index=i;
                 break;
            }
        }
        String ans =num.substring(0,index+1);
    return ans ;
    }
}

File: 2819-remove-trailing-zeros-from-a-string/2819-remove-trailing-zeros-from-a-string.cpp
class Solution {
public:
    string removeTrailingZeros(string num) {
        int index;
        for(int i=num.size()-1;i>=0;i--){
            if(num[i]!='0'){
                 index=i;
                 break;
            }
        }
        string ans =num.substr(0,index+1);
    return ans ;
    }
};